Kitodo-DataEditor/pom.xml

```diff
@@ -56,6 +56,7 @@
         <dependency>
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest</artifactId>
+            <scope>test</scope>
```

Comment:
Not needed as defined in parent as test scope.

Reply:
I only fixed issues reported by the dependency check ...

This also applies to the other locations where I added the test scope as suggested by this check.

Reply:
I get no issues on running different dependency checks. So maybe your tooling is to new? Or how did you get this issues.

Reply:
See commit description: "The missing scopes were reported by `mvn dependency:analyze`."

Reply:
And again: I'm not getting this missing scopes report:
```
[INFO] --- maven-dependency-plugin:2.8:analyze (default-cli) @ kitodo-data-editor ---
Unable to process: org.kitodo.dataeditor.MetsKitodoConverter
Unable to process: org.kitodo.dataeditor.MetsKitodoWrapper
Unable to process: org.kitodo.dataeditor.VersionProvider
Unable to process: org.kitodo.dataeditor.ruleset.KeyView
Unable to process: org.kitodo.dataeditor.ruleset.xml.Ruleset
Unable to process: org.kitodo.dataeditor.ruleset.xml.EditingElement
Unable to process: org.kitodo.dataeditor.ruleset.xml.ConditionsMap
Unable to process: org.kitodo.dataeditor.ruleset.Rule
Unable to process: org.kitodo.dataeditor.ruleset.DivisionDeclaration
Unable to process: org.kitodo.dataeditor.ruleset.RulesetManagement
Unable to process: org.kitodo.dataeditor.ruleset.KeyDeclaration
Unable to process: org.kitodo.dataeditor.ruleset.Labeled
Unable to process: org.kitodo.dataeditor.ruleset.Settings
Unable to process: org.kitodo.dataeditor.ruleset.NestedKeyView
Unable to process: org.kitodo.dataeditor.entities.FileSec
Unable to process: org.kitodo.dataeditor.entities.PhysicalStructMapType
Unable to process: org.kitodo.dataeditor.entities.LogicalStructMapType
Unable to process: org.kitodo.dataeditor.JaxbXmlUtils
Unable to process: org.kitodo.dataeditor.MetsKitodoReader
Unable to process: org.kitodo.dataeditor.DataEditorTest
Unable to process: org.kitodo.dataeditor.MetsKitodoWriterTest
Unable to process: org.kitodo.dataeditor.ruleset.RulesetManagementIT
Unable to process: org.kitodo.dataeditor.MetsKitodoWrapperTest
[WARNING] Used undeclared dependencies found:
[WARNING]    commons-io:commons-io:jar:2.11.0:provided
[WARNING]    org.junit.jupiter:junit-jupiter-api:jar:5.9.2:test
[WARNING] Unused declared dependencies found:
[WARNING]    org.junit.jupiter:junit-jupiter-engine:jar:5.9.2:test
[WARNING]    org.hamcrest:hamcrest:jar:2.2-rc1:test
```